be/src/exprs/agg/percentile_cont.h

```diff
@@ -120,6 +120,7 @@ class PercentileContAggregateFunction final
             result = new_vector[index] + (u - (double)index) * (new_vector[index + 1] - new_vector[index]);
         } else {
             LOG(ERROR) << "Invalid PrimitiveTypes for percentile_cont function";
+            return;
```

Comment:
Should return here when input type is not supported.
You can mark `[[maybe_unused]] ResultType result;`